src/onnx/parse_split.cpp

```diff
@@ -55,12 +55,13 @@ struct parse_split : op_parser<parse_split>
         {
             literal s = parser.parse_value(info.attributes.at("split"));
             s.visit([&](auto v) { vec_splits.assign(v.begin(), v.end()); });
+        }
+        else if(args.size() == 2)
+        {
+            assert(args[1]->can_eval());
```

Comment:
instead of `assert`, i think`MIGRAPHX_THROW` is better. `assert` would only kick if it is debug build.

Reply:
I guess we want to signal that this is invalid onnx, I can change to throw